src/test_typing.py

```diff
@@ -20,6 +20,25 @@
 import typing
 
 
+class TestCase(_TestCase):
+
+    # vanilla TestCase doesn't have assert*Subclass, see bugs.python.org/14819
```

Comment:
Looks like it's not going to happen any time soon. I propose not to count on it. The rename dance TestCase -> _TestCase feels kind of awkward -- why not just pick a decent new name for TestCase + assert[Not]IsSubclass, and use that where it's needed.